Replicator/RevFinder.cc

```diff
@@ -122,7 +122,7 @@ namespace litecore::repl {
                 response.compressed = true;
                 if (!_db->usingVersionVectors()) {
 #if 1
-                    response["maxHistory"_sl] = 20;
+                    response["maxHistory"_sl] = 100;
```

Comment:
Is it possible to get this value from ReplicatorTuning.hh ?

Reply:
You're right, it is.